.github/workflows/tests.yml

```diff
@@ -14,6 +14,7 @@ env:
   HOMEBREW_NO_INSTALL_FROM_API: 1
   HOMEBREW_TEST_BOT_ANALYTICS: 1
   HOMEBREW_ENFORCE_SBOM: 1
+  HOMEBREW_NO_BUILD_ERROR_ISSUES: 1
```

Comment:
Thoughts on setting this in test bot instead?

Reply:
Wouldn't that enforce it for third-party taps too? Do we want that?

Reply:
Is it useful information in the context of `brew test-bot`? I understand its use for end users but `brew test-bot` is a different audience.